about_our_class.md

```diff
@@ -1,9 +1,5 @@
-# About Our Class
-As a class exercise, we will each use git and GitHub to add our own little autobiography to this file!
-
-## Example Name
-Hello, I'm Example Name. 
```

Comment:
If you'd like to have a title-like line for your name, you can put `#` symbols at the start of the line - each `#` indicates a lower-level, more specific sort of header formatting.

For other fun things possible with markdown, check out this [basic markdown formatting guide](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) 

It's just a sort of optional thing you can do to make your section look nicer.